superset-frontend/src/views/CRUD/data/database/DatabaseModal/styles.ts

```diff
@@ -284,85 +280,86 @@ export const validatedFormStyles = (theme: SupersetTheme) => css`
 `;
 
 export const StyledInputContainer = styled.div`
-  margin-bottom: ${({ theme }) => theme.gridUnit * 6}px;
-  &.mb-0 {
-    margin-bottom: 0;
-  }
-  &.mb-8 {
-    margin-bottom: ${({ theme }) => theme.gridUnit * 2}px;
-  }
-
-  .control-label {
-    color: ${({ theme }) => theme.colors.grayscale.dark1};
-    font-size: ${({ theme }) => theme.typography.sizes.s - 1}px;
-    margin-bottom: ${({ theme }) => theme.gridUnit * 2}px;
-  }
+  ${({ theme }) => css`
+    margin-bottom: ${theme.gridUnit * 6}px;
+    &.mb-0 {
+      margin-bottom: 0;
+    }
+    &.mb-8 {
+      margin-bottom: ${theme.gridUnit * 2}px;
+    }
 
-  &.extra-container {
-    padding-top: 8px;
-  }
+    .control-label {
+      color: ${theme.colors.grayscale.dark1};
+      font-size: ${theme.typography.sizes.s - 1}px;
```

Comment:
Do we really need the `-1`? Can't we just change it to `s`?

Reply:
I really don't like "hacking" our theme like that, but I'm kinda nervous about making such changes in this PR... It's not obvious if it should be `s` or `xs`.